ws/ws/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>org.kaazing</groupId>
             <artifactId>specification.ws</artifactId>
-            <version>[0.1,0.99)</version>
+            <version>2.0.0</version>
```

Comment:
Why are we hardcoding the value here instead of using the latest version in the range?  During our release process we will freeze on the latest dependency, so this is an exception to our process.